srcopsmetrics/storage.py

```diff
@@ -97,8 +93,13 @@ class KnowledgeStorage:
     def __init__(self, is_local: Optional[bool] = False):
         """Initialize to behave as either local or remote storage."""
         self.is_local = is_local
+        location = os.getenv(StoragePath.LOCATION_VAR.value)
```

Comment:
`os.getenv` allows you to indicate a default value like this: `os.getenv(<envvar>, default)`

Reply:
nice one! thx!